packages/dapp/components/Layout/Footer.tsx

```diff
@@ -43,9 +43,8 @@ const TextLink = ({ label, href, external }: TextLinkType) => {
     <ChakraLink
       href={href}
       color="main"
-      borderBottom="1px solid"
-      borderBottomColor="main"
-      _hover={{ borderBottomColor: 'white' }}
+      // _hover={{ borderBottomColor: 'white' }}
+      fontFamily="Merriweather"
```

Comment:
we want to use `"body"` here 
and `"heading"` for Gothic Font. Its been setup already in `utils/theme`
So if want to change this font in the future we can easily update it there @rishabnt 

I'm making the changes but letting you know for the future.